### DISCUSSION TITLE
[REQ/IMP] Services - Ratings, Filters, Sorting, or Ability to Hide

### DISCUSSION THREAD

**Discussion by DysektAI** [coollabsio/coolify#3802] [Improvement Requests]
I am VERY overwhelmed with the vast resources of services. As nice as it is to have such great support and wide adoption, it also adds a bit of confusion and disorientation scrolling a list of services. The docs is similar in this respect a bit more informative, but overall I would love to know stats for example: Reviews|Ratings, Filter by Usage, Hide or disable certain known services maybe in a "Show more|hidden" menu as WordPress personally I have no interest in and it just clutters the list more and more.

Overall I propose somewhat like a market place style setup, where users can rate, see install thru Coolify, etc. Just basic anonymous stats to help others like myself see what is trending and what is most popular and commonly utilized so I don't waste time checking each website or docs to see what it is exactly and if it applies to me.

**Comment by renequez** [comment]
+1, i was just thinking about this. Even just filtering by category would be nice. With the addition of all the services lately and the ones that follow, it's just a long list of apps, which is hard to follow.
